### DISCUSSION TITLE
Full rewrite

### DISCUSSION THREAD

**Discussion by prevter** [Prevter/GDOpenHack#74] [Announcements]
Check [full changelog](https://github.com/Prevter/GDOpenHack/blob/lemonade/CHANGELOG.md) to read about all changes.

Main changes:
- Project was remade from scratch, should contain less bugs (tm)
- Improved UI and animations with even more customization options
- Added cache for pattern scanner to improve loading times
- Added "Audio Speedhack"
- Added "Physics Bypass"
- Added "Apply 4GB patch"
- Added "Instant Complete"
- and 11 other hacks

Note:
- Discord RPC is temporarily cut from the mod (needs a full rewrite aswell)
- Vanilla version doesn't support versions other than 2.204 for now (will be fixed later as well)
- Many cool things are planned for 2.1.0, so stay tuned ;)

<hr /><em>This discussion was created from the release <a href='https://github.com/Prevter/GDOpenHack/releases/tag/2.0.0'>Full rewrite</a>.</em>

**Comment by Rustring** [comment]
Oh that's why it took so long for an update.

**Reply by Bituvo** [reply]
giggity giggity
